pkg/columns/columns.go

```diff
@@ -198,6 +198,15 @@ func (c *Columns[T]) iterateFields(t reflect.Type, sub []int) error {
 			return fmt.Errorf("error parsing tag for %q on field %q: %w", t.Name(), f.Name, err)
 		}
 
+		// fall back to struct field name if column name is empty
+		if column.Name == "" {
+			column.Name = f.Name
+		}
```

Comment:
When could this happen?

Reply:
If you define `Field string column:",width:20"` for example.  In this case the name then would become `Field`.

Reply:
Mmmm not sure, I would prefer to return an error also here.

Reply:
It's the same way that `encoding/json` handles that case - you can specify `json:",omitempty"` and it'll do the same.

Reply:
And what happens if more than one field doesn't specify the name?

Reply:
They all get their respective field names - if you used that field name before for another field, it'll return the "duplicate name" error.